README.md

```diff
@@ -834,8 +834,6 @@ ANSWERS.addComponent('VerticalResults', {
   // Optional, a modifier that will be appended to a class on the results list like this `yxt-Results--{modifier}`
   modifier: '',
   // Optional, whether to hide the default results header that VerticalResults provides. Defaults to false.
-  hideResultsHeader: false,
-  // Optional, the card used to display each individual result, see the Cards section for more details,
```

Comment:
It looks like you deleted the comment below the option rather than the comment above the option

Reply:
ugh my bad